src/render_tasks/d3d12_rtao_task.hpp

```diff
@@ -204,10 +211,10 @@ namespace wr
 				auto camera = scene_graph.GetActiveCamera();
 				temp::RTAO_CBData cb_data;
 				cb_data.m_inv_vp = DirectX::XMMatrixInverse(nullptr, camera->m_view * camera->m_projection);
-				cb_data.bias = 0.01f;
-				cb_data.radius = 1.f;
-				cb_data.power = 1.f;
-				cb_data.sample_count = 8u;
+				cb_data.bias = rtaoSettings::bias;
+				cb_data.radius = rtaoSettings::radius;
+				cb_data.power = rtaoSettings::power;
+				cb_data.sample_count = static_cast<unsigned int>(rtaoSettings::sample_count);
```

Comment:
ImGui doesn't work with unsigned int so we save this as int and static_cast it before passing it to the CB